pkg/client/e2e.go

```diff
@@ -75,8 +75,7 @@ func (*SonobuoyClient) GetTests(reader io.Reader, show string) ([]reporters.JUni
 // Focus returns a value to be used in the E2E_FOCUS variable that is
 // representative of the test cases in the struct.
 func Focus(testCases []reporters.JUnitTestCase) string {
-	// YAML doesn't like escaped characters and regex needs escaped characters. Therefore a double escape is necessary.
-	r := strings.NewReplacer("[", `\\[`, "]", `\\]`)
+	r := strings.NewReplacer("[", `\[`, "]", `\]`)
```

Comment:
The fact that this escaping changed may be a result of how we changed some yaml serialization. We used to manually create the plugins but I modified that to just use the go-yaml library so I think we changed how we were quoting things, therefore probably how things needed to be internally escaped.